langserver/handler.go

```diff
@@ -72,6 +73,13 @@ type LangHandler struct {
 
 // reset clears all internal state in h.
 func (h *LangHandler) reset(init *InitializeParams) error {
+	for _, k := range init.Capabilities.TextDocument.Completion.CompletionItemKind.ValueSet {
+		if k == lsp.CIKConstant {
+			CIKConstantSupported = lsp.CIKConstant
```

Comment:
you are setting a global variable here, but langserver supports multiple instances running at once in the same process.